universal-application-tool-0.0.1/app/services/applicant/question/IDQuestion.java

```diff
@@ -0,0 +1,120 @@
+package services.applicant.question;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSet;
+import java.util.Optional;
+import services.MessageKey;
+import services.Path;
+import services.applicant.ValidationErrorMessage;
+import services.question.types.IDQuestionDefinition;
+import services.question.types.QuestionType;
+
+/**
+ * Represents an id question in the context of a specific applicant.
```

Comment:
suggestion (non-blocking): maybe give a couple examples of IDs e.g. library account number?